bin/plugin/commands/performance.js

```diff
@@ -260,6 +261,22 @@ async function runPerformanceTests( branches, options ) {
 	);
 
 	log( '>> Starting the WordPress environment' );
+	if ( options.wpVersion ) {
+		// Patch the environment's .wp-env.json config to use the specified WP version:
+		//
+		//     {
+		//         "core": "https://wordpress.org/wordpress-$VERSION",
+		//         ...
+		//     }
+		//
+		// Only use the major version, i.e. 5.7.2 becomes 5.7.
+		const major = options.wpVersion.split( '.' ).slice( 0, 2 ).join( '.' );
```

Comment:
How about leaving this to the caller? I think it's arguably not this script's responsibility (plus we're not really documenting it in `bin/plugin/cli.js`).

Reply:
This was me taking a shortcut because of how ZIP files are named on the server. See 1217cff7aab08c6c864190aca64cc9ed64ac6e05, in which I did things correctly. :)